package-lock.json

```diff
@@ -1,16 +1,17 @@
 {
-  "name": "@solana/web3.js",
-  "version": "0.0.0-development",
+  "name": "@exodus/solana-web3.js",
```

Comment:
why do we need this file at all?

Reply:
This is automatically generated by npm.

Reply:
yes, but can we just delete it from git?

Reply:
We need this file mostly for testing purposes, I think? Upstream uses `package-lock.json`, there doesn't seem to be a good reason to convert that to `yarn.lock` or to not use any lockfile at all.

Reply:
@mvayngrib we definitely need it as we create a bundle with the deps.

Reply:
ah, didn't realize that